x-pack/legacy/plugins/apm/public/components/shared/Stacktrace/Stackframe.tsx

```diff
@@ -54,16 +55,20 @@ export function Stackframe({
     );
   }
 
+  const { lines, highlightRanges, lineMapper } = formatLinesForCodeblock(
+    stackframe
+  );
+
   return (
     <Container isLibraryFrame={isLibraryFrame}>
       <CodeHeader>
```

Comment:
@rylnd can you suggest on how to embed this `CodeHeader` and  `Variables` into the `CodeBlock` component?

Reply:
I think for now, simply making `CodeHeader` and `Variables` siblings like this means less responsibility for `CodeBlock`, and an identical amount of work for the consumer. Within Code, we're going to continue to use `CodeBlockPanel` which accepts a header prop, but even that is of dubious value.

Reply:
sg